openrouteservice/src/main/java/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -233,6 +245,61 @@ protected void cleanUp() {
         super.cleanUp();
     }
 
+    public GHResponse directRoute(GHRequest request) {
```

Comment:
should be named better to describe what the method actually does

Reply:
I renamed it to constructAirLineRoute...

Reply:
still sounds a bit weird to me... I will update it to constructFreeHandRoute as the term air line is not used anywhere else and so it is a bit confusing